config/database.yml

```diff
@@ -1,25 +1,16 @@
-# SQLite version 3.x
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
-#
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  encoding: unicode
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: <%= Rails.application.secrets["database_url"] %>
```

Comment:
Per [the Rails documentation](http://guides.rubyonrails.org/configuring.html#configuring-a-database), the `DATABASE_URL` URI should use the `url` key _(vs. `database`)_ in the `config.yml`. We should also enable easy use of the URL format consistently across environments vs. just in `development`, as this is required for deployment to 12-factor environments such as Heroku.

Reply:
cool, thanks for the link to the docs ( should have been the first place I looked :) ).

So the Fix ought to just be switching the line to ```url: <%= Rails.application.secrets["database_url"] %> ```  ?

Reply:
> So the Fix ought to just be switching the line to `url: <%= Rails.application.secrets["database_url"] %>`?

Yes, but I think Rails is now preferring Symbols to strings for the secrets:

```ruby
Rails.application.secrets[:database_url]
```